examples/FirebaseDemo_ESP8266/FirebaseDemo_ESP8266.ino

```diff
@@ -20,11 +20,19 @@
 #include <ESP8266WiFi.h>
 #include <FirebaseArduino.h>
 
-void setup() {
-  Serial.begin(9600);
+// Set these to run example.
+#define FIREBASE_HOST "example.firebaseio.com"
+#define FIREBASE_AUTH "token_or_secret"
+#define WIFI_SSID "SSID"
+#define WIFI_PASSWORD "PASSWORD"
+
+void ConnectWifi(const String& ssid, const String& password = "") {
```

Comment:
I would support adding this to a header and just importing it into all of these examples. The user doesn't really need to know what is happening here.